login/FrontQuery.js

```diff
@@ -0,0 +1,7 @@
+class FrontQuery{
+    constructor(login,password,name,surname,id,rankId,date){
+        this.login=login;
```

Comment:
You sure you want to pass the login to the frontend instead of user's Id? (User's Id is by default indexed[provides logarithmic{very fast} time complexity{search speed}]), so if you don't need to display the user's name on the frontend, then maybe it's better to switch it out, but if you don't I'd still pass the id there too for better search speed on user's Id.

Reply:
ok,we need login on front but i will add id